bundles/automation/org.eclipse.smarthome.automation.module.script.rulesupport/src/main/java/org/eclipse/smarthome/automation/module/script/loader/internal/file/ScriptFileProvider.java

```diff
@@ -0,0 +1,217 @@
+/**
```

Comment:
Is it really used? It is not even in the correct namespace for this bundle, so it seemed to be some left-over to me...